src/main/webapp/resources/js/pages/password/PasswordReset.jsx

```diff
@@ -0,0 +1,242 @@
+          <Alert
+            style={{ marginBottom: SPACE_SM }}
+            className="t-reset-error-alert"
+            message="The following errors occurred:"
+            description={
+              <Typography.Paragraph>
+                {
+                  <List>
+                    {Object.keys(errorMessages).map(function (keyName) {
+                      return (
+                        <List.Item key={keyName}>
+                          {errorMessages[keyName]}
+                        </List.Item>
+                      );
+                    })}
+                  </List>
+                }
+              </Typography.Paragraph>
+            }
+            type="error"
+            showIcon
+          />
+        )}
+
+        {!updateSuccess
+          ? passwordExpired && (
+              <Alert
+                style={{ marginBottom: SPACE_SM }}
+                className="t-password-expired-alert"
+                message={i18n("password.reset.password_expired")}
+                type="error"
+                showIcon
+              />
+            )
+          : null}
+
+        {!updateSuccess && (
+          <div>
+            <Alert
+              style={{ marginBottom: SPACE_SM }}
+              message={i18n("PasswordReset.alert.title")}
+              className="t-password-policy-alert"
+              description={
+                <Typography.Paragraph>
+                  <List
+                    header={i18n("PasswordReset.alert.description")}
+                    dataSource={passwordRules}
+                    renderItem={(item) => <List.Item>- {item}</List.Item>}
+                  />
+                </Typography.Paragraph>
+              }
+              type="info"
+              showIcon
+            />
+            <Form
+              form={passwordResetForm}
+              name="resetPasswordForm"
+              size="large"
+              onFinish={submitPasswordResetForm}
+            >
+              <Item
+                name="password"
+                rules={[
+                  ({}) => ({
+                    validator(_, value) {
+                      return validatePassword(value);
+                    },
+                  }),
+                ]}
+              >
+                <Input.Password
+                  name="password"
+                  id="password"
+                  ref={passwordRef}
+                  prefix={<LockOutlined style={{ color: blue6 }} />}
+                  placeholder={i18n("PasswordReset.input.placeholder")}
+                  disabled={loading}
+                  iconRender={(visible) =>
+                    visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />
+                  }
+                />
+              </Item>
+              <Item>
+                <Button
+                  className="t-submit-btn"
+                  type="primary"
+                  disabled={loading || invalidPassword}
+                  icon={loading && <LoadingOutlined />}
```

Comment:
Same here, use `loading` attr

Reply:
Updated in eb461f960fdd78a45922a9339cbc750424d85595